Formula/apache-archiva.rb

```diff
@@ -4,17 +4,28 @@ class ApacheArchiva < Formula
   url "https://www.apache.org/dyn/closer.lua?path=archiva/2.2.5/binaries/apache-archiva-2.2.5-bin.tar.gz"
   mirror "https://archive.apache.org/dist/archiva/2.2.5/binaries/apache-archiva-2.2.5-bin.tar.gz"
   sha256 "01119af2d9950eacbcce0b7f8db5067b166ad26c1e1701bef829105441bb6e29"
-  license "Apache-2.0"
+  license all_of: ["Apache-2.0", "GPL-2.0-only"]
+  revision 1
 
   bottle do
     sha256 cellar: :any_skip_relocation, all: "7be56a181bec0957d8f4f2936e99a98f737b3f7e146a3f7384b13d9e824919ad"
   end
 
+  depends_on "ant" => :build
+  depends_on "java-service-wrapper"
   depends_on "openjdk"
 
   def install
     libexec.install Dir["*"]
+    rm_f Dir["#{libexec}/bin/wrapper*"]
+    rm_f Dir["#{libexec}/lib/libwrapper*"]
     (bin/"archiva").write_env_script libexec/"bin/archiva", JAVA_HOME: Formula["openjdk"].opt_prefix
+
+    wrapper = Formula["java-service-wrapper"].opt_libexec
+    ln_s wrapper/"bin/wrapper", "#{libexec}/bin/wrapper"
+    libext = OS.mac? ? "jnilib" : "so"
+    ln_s wrapper/"lib/libwrapper.#{libext}", "#{libexec}/lib/libwrapper.#{libext}"
+    ln_s wrapper/"lib/wrapper.jar", "#{libexec}/lib/wrapper.jar"
```

Comment:
```suggestion
    ln_s (wrapper/"bin/wrapper").relative_path_from(libexec/"bin"), libexec/"bin"
    libext = OS.mac? ? "jnilib" : "so"
    ln_s (wrapper/"lib/libwrapper.#{libext}").relative_path_from(libexec/"lib"), libexec/"lib"
    ln_s (wrapper/"lib/wrapper.jar").relative_path_from(libexec/"lib"), libexec/"lib"
```
We want to make sure we create relative symlinks so this is reloctable. (`install_symlink` is not good for symlinks to other kegs.)

Though, maybe `brew` does this automatically when bottling; I don't remember. I can have a look at the bottle when it gets built.